README.md

```diff
@@ -126,7 +124,7 @@ argnorm abricate --db resfinder -i examples/hamronized/abricate.resfinder.tsv -o
 #### Raw
 ```bash
 argnorm abricate --db ncbi -i examples/raw/abricate.ncbi.tsv -o outputs/raw/abricate.ncbi.tsv
-argnorm abricate --db megares -i examples/raw/abricate.megares.tsv -o outputs/raw/abricate.megarest.tsv
```

Comment:
Still needs fixing

Reply:
I am not too sure about which part you are referring to here. Isn't 'megarest' being replaced by 'megares' as intended?

Reply:
yeah, sorry